src/parser/Debug_Options.cc

```diff
@@ -12,10 +12,67 @@
 
 #include "Debug_Options.hh"
 
```

Comment:
It's been my practice with Class.cc, Class.hh file pairs to have the include directive for Class.hh come first in Class.cc. This ensures that there will be an error when Class.cc is compiled if some include file or using statement that should be in Class.hh (to make it self-contained) is missing.